pkg/features/kube_features.go

```diff
@@ -434,6 +433,12 @@ const (
 	//
 	// Enables the OpenStack Cinder in-tree driver to OpenStack Cinder CSI Driver migration feature.
 	CSIMigrationOpenStack utilfeature.Feature = "CSIMigrationOpenStack"
+
+	// owner: @verult
+	// GA: v1.13
+	//
+	// Enables the regional PD feature on GCE.
+	gceRegionalPersistentDisk utilfeature.Feature = "gceRegionalPersistentDisk"
```

Comment:
The string value still has to be `GCERegionalPersistentDisk`

Reply:
fixed.